.github/workflows/pypi_publish.yml

```
@@ -0,0 +1,14 @@
+run-name: publish to pypi
+
+on:
+  workflow_dispatch:
+
+jobs:
+  wait-for-approval:
+    uses: datachecks/github-action-workflows/.github/workflows/manual_approver.yaml@main
+
+  publish-to-pypi:
+    needs: wait-for-approval
+    uses: datachecks/github-action-workflows/.github/workflows/poetry_publish.yaml@main
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)